src/Microsoft.ML.Data/StaticPipe/Transformer.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+
+namespace Microsoft.ML.Data.StaticPipe
+{
+    public sealed class Transformer<TTupleInShape, TTupleOutShape, TTransformer>
+        where TTransformer : class, ITransformer
+    {
+        private readonly IHostEnvironment _env;
+        public TTransformer Wrapped { get; }
```

Comment:
>Wrapped [](start = 28, length = 7)

maybe rename these things to be `Estimator`, `Transformer` and `Data` ?

Reply:
`.AsDynamic` ?

---
In reply to: [214470782](https://github.com/dotnet/machinelearning/pull/778#discussion_r214470782) [](ancestors = 214470782)